relay-config/src/config.rs

```diff
@@ -1014,6 +1014,9 @@ pub struct Processing {
     /// Maximum rate limit to report to clients.
     #[serde(default = "default_max_rate_limit")]
     pub max_rate_limit: Option<u32>,
+    /// Granularity of normalization this event should do.
+    #[serde(default)]
+    pub normalize_events: NormalizeEvents,
```

Comment:
Should we strip the `_events` suffix here? We are normalizing more and more non-event data items, and I guess the logic should (eventually) apply regardless of the data type?

```suggestion
    pub normalize: Normalize,
```